uber/models.py

```diff
@@ -1123,16 +1123,16 @@ def badge_cost(self):
 
     @property
     def age_group_conf(self):
```

Comment:
This function hadn't been well tested after the config changes I made this year :(

It's all working now though :)